phpBB/styles/prosilver/template/navbar_header.html

```diff
@@ -192,7 +192,7 @@
 			{% EVENT overall_header_breadcrumb_prepend %}
 				<span class="crumb" {{ MICRODATA }}><a itemprop="item" href="{{ U_INDEX }}" accesskey="h" data-navbar-reference="index">{% if not U_SITE_HOME %}<i class="icon fa-home fa-fw"></i>{% endif %}<span itemprop="name">{{ L_INDEX }}</span></a><meta itemprop="position" content="{{ navlink_position }}{% set navlink_position = navlink_position + 1 %}" /></span>
 
-			{% for navlinks in navlinks %}
+			{% for navlinks in loops.navlinks %}
```

Comment:
Since no `loops.` are to be used in the codebase, will

```
{% if navlinks|length %}
	{% for navlinks in navlinks %}
		{% set NAVLINK_NAME .....
	{% endfor %}
{% endif %}
```
do the trick here?

Reply:
Did not know `loops.` was to be avoided.
I'll update it then.

Reason I went for this in the first place is cause I am not a fan of using the same variable as the loop name, so this would solve both at once. This would be the equivalant in PHP of `foreach ($navlinks as $navlinks)` which is wrong, so I would that rather change it to `{% for navlink in navlinks %}` with the `{% if %}` statement surrounding it.

Reply:
Yeah, in this case was possible to change the variable name, in some other case it wasn't.